surveys/2022-annual-survey/questions.md

```diff
@@ -26,12 +26,65 @@ Type: select one
 
 ## For previous Rust users
 
+### Why did you stop using Rust?
+
+Type: select all that apply
+
+- Rust did not help me achieve my goals
+- Missing language features
+- Missing libraries
+- Missing tools
+- Too difficult to learn
+- Community was rude, unwelcoming, or otherwise off-putting
+- I prefer to use another language
+- I no longer have the opportunity to use Rust due to external factors
+- Other
+
+### Tell us more:
+
+Type: free text
+
+### Would you be likely to use Rust if the above issues were addressed
+
+Type: select one
+
+- Yes
+- No
+
 > **SURVEY FLOW**
 >
 > Skip to `## Your opinions about Rust` section
 
+TODO there are some questions before this which we could ask non-users, how do we make this work with the software?
+
 ## For non-Rust users
 
+### Why don't you use Rust?
+
+Type: select all that apply
+
+- Rust did not help me achieve my goals
+- Missing language features
+- Missing libraries
+- Missing tools
+- Too difficult to learn or learning will take too much time
+- Community was rude, unwelcoming, or otherwise off-putting
+- I prefer to use another language
+- I no longer have the opportunity to use Rust due to external factors
```

Comment:
I don't think this makes sense in this question - and definitely not how it's currently worded.